packages/server/src/__tests__/standalone/standaloneSpecific.test.ts

```diff
@@ -103,3 +103,41 @@ describe('Typings: TContext inference', () => {
     await server.stop();
   });
 });
+
+describe('Configuration', () => {
+  it('allows > 100KiB bodies to be sent (body-parser default)', async () => {
+    const server = new ApolloServer({
+      typeDefs: `type Query { hello: String }`,
+      resolvers: {
+        Query: {
+          hello: () => 'hello world!',
+        },
+      },
+    });
+
+    const { url } = await startStandaloneServer(server, {
+      listen: { port: 0 },
+    });
+
+    const excessivelyLargeBody = JSON.stringify({
+      query: `{hello}`,
+      variables: Object.fromEntries([...Array(7960)].map((_, i) => [i, 'foo'])),
```

Comment:
GraphQL doesn't require all variable values to match declared variables, but it's maybe a little awkward that these variables don't even have valid names (they start with a digit)...  I mean, not "wrong" but you could also make it actually valid pretty easily (can just be one very long string or one very long array of ints too).

Reply:
[93d24a4](https://github.com/apollographql/apollo-server/pull/7172/commits/93d24a408438da81e1d5088fd79d54f3024063f7)